idea-plugin/src/main/java/com/palantir/javaformat/intellij/PalantirJavaFormatFormattingService.java

```diff
@@ -89,9 +93,20 @@ public void run() {
             }
 
             try {
-                String formattedText = applyReplacements(
-                        request.getDocumentText(),
-                        formatterService.get().getFormatReplacements(request.getDocumentText(), toRanges(request)));
+                logger.info(String.format(
```

Comment:
example logs: 
```
2025-04-04 16:06:38,464 [ 339449]   INFO - #com.palantir.javaformat.intellij.PalantirJavaFormatFormattingService - Received request to format file=/Volumes/git/<file>.java, length=10912 with ranges=[(0,10912)]
2025-04-04 16:06:38,590 [ 339575]   FINE - #com.palantir.javaformat.intellij.PalantirJavaFormatFormattingService - Applying 1 replacements with ranges=[[0..10912)]
```

Reply:
is info not noisy or does it not matter?

Reply:
changed this to DEBUG level. 
Initially I wanted this to be logged all the time in case someone else is hitting this issue, but I guess it is fine to enable debug once someone is able to repro the issue.